.openshift-ci/drivers/scripts/kernel-splitter.py

```diff
@@ -128,25 +143,31 @@ def split(self, new_builders: int) -> list:
             })
 
         return [
-            Builder(f'{self.name}/{i}', self.regex, tasks)
+            Builder(f'{self.name}/{i}', self.regex, tasks, self.ebpf_only, self.ko_only)
             for i, tasks in enumerate(new_builders_tasks)
         ]
 
 
 def main(task_file):
-    fc36_kernels = r'(?:5\.[1-9]\d+\..*)'
-    rhel8_kernels = r'(?:(?:4|5)\.\d+\..*)'
-    rhel7_kernels = r'(?:3\.\d+\..*)'
-
-    fc36 = Builder('fc36', fr'^{fc36_kernels}', {})
-    rhel8 = Builder('rhel8', fr'^{rhel8_kernels}', {})
-    rhel7 = Builder('rhel7', fr'^{rhel7_kernels}', {})
-    unknown = Builder('unknown', r'.*', {})
+    fc36_kernels = r"(?:5\.[1-9]\d+\..*)"
+    rhel8_kernels = r"(?:(?:4|5)\.\d+\..*)"
+    rhel7_kernels = r"(?:3\.\d+\..*)"
+    # rhel8_ebpf_kernels = r'(?:5\.\d+\..*)'
+    rhel7_ebpf_kernels = r"(?:(?:3|4|5)\.\d+\..*)"
+
+    fc36 = Builder("fc36", rf"^{fc36_kernels}", {})
+    rhel8 = Builder("rhel8", rf"^{rhel8_kernels}", {})
+    # rhel8_ebpf = Builder('rhel8', fr'^{rhel8_ebpf_kernels}', {}, ebpf_only=True)
+    rhel7 = Builder("rhel7", rf"^{rhel7_kernels}", {})
+    rhel7_ebpf = Builder("rhel7", rf"^{rhel7_ebpf_kernels}", {}, ebpf_only=True)
+    unknown = Builder("unknown", r".*", {})
 
     builders = [
-        fc36,
-        rhel8,
-        rhel7,
+        fc36,  # 5.10+ any
+        # rhel8_ebpf, # 5+ ebpf
+        rhel7_ebpf,  # 3+/4+/5.0-5.9 ebpf
+        rhel8,  # 4/5+ any
+        rhel7,  # 3+ any
```

Comment:
Never mind this one, it would make the dumping stage a little messy probably.

Reply:
because of the reconstruction of this list later with the sharded builders, it's a little tricky to initialise everything in-line, but I've adjusted the order of the builder declarations to match the order in the array